kicksaw_integration_utils/aws/sqs/queue.py

```diff
@@ -0,0 +1,339 @@
+from __future__ import annotations
```

Comment:
```suggestion
```

not needed, right?

Reply:
This wouldn't work then https://github.com/Kicksaw-Consulting/kicksaw-integration-utils/blob/f8a4617cbb77df96528c73644fce2248715ee471/kicksaw_integration_utils/aws/sqs/queue.py#L67-L74

AFAIK, posponed annotations are still not a thing even though they planned them for `3.10`: https://docs.python.org/3/library/__future__.html#id1 As long as we support python versions without this (currently these are all versions) we need this